cigaradvisor/styles/styles.css

```diff
@@ -248,7 +248,6 @@ main .section[data-layout] .layout-wrapper > div {
   display: flex;
   flex-direction: column;
   align-items: stretch;
-  justify-content: space-between;
```

Comment:
This has more implications than just fixing the CTA wrapper

Reply:
modified. pls review.